test/e2e/devconsole.operatorsource.cr.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: devconsole-operator
+  namespace: openshift-operators
+spec:
+  type: appregistry
+  endpoint: https://quay.io/application/ldimaggi-test-org/mychart
```

Comment:
According to the [official example](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/community.operatorsource.cr.yaml) the 
[`spec.endpoint`](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/community.operatorsource.cr.yaml#L5) should be `https://quay.io/cnr`
and [`spec.registryNamespace`](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/community.operatorsource.cr.yaml#L8) should be `ldimaggi-test-org`